### DISCUSSION TITLE
URLs under subdirectory display 404 with [rewrites]

### DISCUSSION THREAD

**Discussion by danielunited** [vercel/community#1508] [Help]
### Summary

Hi guys,

I've followed [this article](https://vercel.com/guides/how-can-i-serve-multiple-projects-under-a-single-domain) to try & setup multiple projects under the same domain. Here's what my `vercel.json` file looks like:

```
{
  "rewrites": [
    { "source": "/create", "destination": "https://create-app.vercel.app/" },
    { "source": "/create/:match*", "destination": "https://create-app.vercel.app/:match*" },
    { "source": "/blog", "destination": "https://blog-app.vercel.app/" },
    { "source": "/blog/:match*", "destination": "https://blog-appvercel.app/:match*" },
    { "source": "/", "destination": "https://website-app.vercel.app/" },
    { "source": "/:match*", "destination": "https://website-app.vercel.app/:match*" }
  ]
}
```

/create, /blog look good. But navigating files under them, such as `index.js` returns 404 because it's trying to access `example.com/index.js` instead of `example.com/create/index.js`

Create is a Nuxt application and I did setup:  `router: { base: "/create/"}` there but it doesn't help.

What do you think is the issue?

**Comment by amyegan** [comment]
Hi @danielunited! Do you have an example deployment link or minimal reproduction that I can use to get a better look at the problem?

**Comment by danielunited** [comment]
I've resolved this by settings up a base directory (within the project) and `PREFIX_PATHS = true` on Vercel's deployment :)

**Reply by heikopaiko** [reply]
Hey @danielunited would you mind elaborating a bit on how you actually got it working? e.g. where did you add the prefix etc.?
I am stuck on this for quite some time and its getting a bit frustrating ...
